util/network/src/client_version.rs

```diff
@@ -16,24 +16,24 @@
 
 #![warn(missing_docs)]
 
-//! Parse ethereum client ID strings and provide querying functionality
+//! Parse Ethereum client ID strings and provide querying functionality
 
 use semver::Version;
 use std::fmt;
 
-/// Parity client string prefix
+/// OpenEthereum client string prefix
 const LEGACY_CLIENT_ID_PREFIX: &str = "Parity-Ethereum";
-const CURRENT_CLIENT_ID_PREFIX: &str = "Open-Ethereum";
+const CURRENT_CLIENT_ID_PREFIX: &str = "OpenEthereum";
```

Comment:
this change will make current master incompatible with further releases and is only possible because we haven't released a new version yet, otherwise we'd have to support three client names

Reply:
I mean current master (not the one after the PR is merged) won't be able to correctly detect client prefix.

Reply:
ok, do we need `LEGACY_CLIENT_ID_PREFIX_2: &str  = "Open-Ethereum"`?

Reply:
> and is only possible because we haven't released a new version yet

No, the last release version has `Parity-Ethereum` as the client prefix, so it's fine.